querybook/webapp/ui/Modal/Modal.scss

```diff
@@ -5,8 +5,8 @@
     }
 
     .Modal-top {
-        font-family: var(--font-title);
         .Modal-title {
+            font-family: var(--font-title);
```

Comment:
can you keep it on top so it applies to other things in modal top like tabs? lets just force all buttons to be font-sans-serif